integration/tests/cook/test_basic.py

```diff
@@ -124,6 +134,11 @@ def test_mea_culpa_retries(self):
 
         uuid, resp = util.submit_job(self.cook_url, command='sleep 30', env={'EXECUTOR_TEST_EXIT': '1'})
         try:
+            instance = util.wait_for_instance(self.cook_url, uuid)
+            if instance['executor'] != 'cook':
```

Comment:
I think this test should have a `skipIf` annotation as there is no point in running the test if Cook executor is not in use.

Reply:
Done